requirements.txt

```diff
@@ -1,8 +1,23 @@
+certifi==2018.11.29
+chardet==3.0.4
+coreapi==2.3.3
+coreschema==0.0.4
 Django==2.1.4
 django-cors-middleware==1.3.1
 django-extensions==2.1.4
+django-rest-swagger==2.1.0
 djangorestframework==3.9.0
-PyJWT==1.7.1
-six==1.12.0 
+idna==2.8
+itypes==1.1.0
+Jinja2==2.10
+MarkupSafe==1.1.0
+openapi-codec==1.3.2
```

Comment:
Do we need all these new packages you added?

Reply:
Let me have a look at them